### ISSUE TITLE
Version 9.6.16 reports 9.6.15 as version

### ISSUE THREAD

**Issue by timsutton** [schollz/croc#703]
Hello, part-time Homebrew core maintainer here, and noticed this when bumping the formula for `croc`:

## Describe the bug

The new croc 9.6.16 version doesn't accurately report the version via `croc --version`:

```
➜  croc_v9.6.16_macOS-ARM64 ./croc --version
croc version v9.6.15
```

## To Reproduce

```
$ cd /tmp
$ curl -L https://github.com/schollz/croc/releases/download/v9.6.16/croc_v9.6.16_macOS-ARM64.tar.gz | tar -xzf -
$ ./croc --version
```

## Expected behaviour
```
croc version v9.6.16
```
## Version

(not going to confuse things here :)

## Additional context
<-- Add any other context about the problem here. -->

**Comment by schollz** [comment]
sorry about that, should be fixed in 9.6.17

**Comment by timsutton** [comment]
Thanks!
